### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by TekyRey**
### **Leaderboard: hit the API**
In this project I :

- Created a new game
- Add posting scores functionality
- Add Fetching data and displaying on a table functionality
- Add refresh button functionality
- Added minimum styling 
**### To start the project, run npm start**

### CODE REVIEW

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @TekyRey  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
alt
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors 💯 
- Nice use of Github  workflow 👍 


## Required Changes ♻️
_Check the comments under the review._

- [ ] Please try to make your work app as expected, it seems that you forgot to link your `index.js` to your `index.html` and the app can't do anything without that .so you can link it to make your app functional.you got what it takes 👍 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mugishasam123** [comment]
hello @TekyRey, sorry for mistakenly requesting changes to your project. Later, I found that it was due to my browser bugging me. Now your project is approved and you can wait and see it reflected in your dashboard.

File: src/index.js [hunk 1]
```diff
@@ -1,13 +1,60 @@
...
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(['', ''], ' ');
+const gameId = 'ngRRYTtqlgOlpU5S061T';
 
-  return element;
-}
+const submitScore = async (userName, userScore) => {
+  const response = await fetch(
+    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`,
```
Comment (BertrandConxy) [inline src/index.js:12]: - [ ] I really have tried all the ways in order to run your application on my machine but none is working. I really do not know how you run your application. If you run it in a special way, please specify it in your README and add the steps to run it 😃 💡. I tried the usual way of starting the webpack dev server by running `npm start` and it was not working. At first, it showed an error that the webpack dev server is not installed. I installed it. I run it again and it kept on showing the same error. I really don't know the cause up to now. Please try to fix it. However, if you run your application in a new way and it is working, please provide the information in both README and the Pull Request description. Here is the screenshot showing error 👇 
image

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @TekyRey ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights 💪 
- no linter errors
- correct GitHub flow is used

 ## Required Changes ♻️

- [ ] After installing your app on my PC. I tried to run `npm start` and I was getting an error telling me that webpack-dev-server is not installed. Please install webpack-dev-server so that everyone doesn't need to install it on their machine 👍

_Check other comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — APPROVED** [review]
Hi @TekyRey, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
 - [X] Correct git flow practice :heavy_check_mark:
 - [X] Good commit messages :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:

## Optional suggestions

- [ ] Please remove the live version link from the README until you deploy the project in order not to lead users to a broken site;

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**